docs/free-trial.rst

```diff
@@ -0,0 +1,103 @@
+.. _free-trial:
+
+==========
+Free trial
+==========
+
+If you want to try out CrateDB Cloud's potential, there are multiple ways to do
+this. One is to subscribe to one of the CrateDB Cloud offers via our SaaS
+marketplace offers and to choose the `Development subscription plan`_. For a
+full guide on how to subscribe via the marketplace offers, see our
+:ref:`tutorials on deploying a cluster from scratch <sign-up>` above.
+
+The other is to avail yourself of the possibility of our free trial: a fully
+functioning CrateDB Cloud cluster running for 14 days without cost, so you can
+experience the benefits of CrateDB Cloud without any immediate commitment.
+
+In this guide, we describe step by step how to activate the CrateDB Cloud free
+trial offer. The CrateDB Cloud structure is based on organizations, and within
+these organizations run projects that contain the deployed clusters. To deploy
+your free trial cluster, you therefore need an organization and a project to
+deploy it in as well. Fortunately, these are created quite easily as part of
+the deployment process, as will be shown in the following steps.
+
+
+.. _free-trial-signup:
+
+Sign up
+=======
+
+In order to access CrateDB Cloud, you must first sign up with an account.
+Simply follow the steps in the :ref:`tutorial for signing up <sign-up>`. When
+you have successfully signed up, you will be forwarded to the CrateDB Cloud
+Console subscription overview. It is from here that you can make use of the
+free trial option. The first step is to set up an organization.
+
+
+.. _free-trial-org:
+
+Create an organization
+======================
+
+Before the free trial can be deployed, it is necessary to create an
+organization within the CrateDB Cloud Console. The trial cluster is then
+deployed within this organization. When you arrive for the first time at the
+Console subscription overview, you should be prompted to create an
+organization.
+
+.. image:: _assets/img/free-trial-organization.png
+
+Enter the desired name for the organization in the field and click *Create
+organization*. Once you have done so, you will see in the subscription overview
+screen a set of options for choosing a cloud provider: Microsoft Azure, Amazon
+AWS, and the free trial cluster option.
+
+.. image:: _assets/img/free-trial-link.png
+
+You can now deploy the free trial cluster by clicking the *Launch free 14-day
+trial cluster* button. This will take you to the configuration wizard for the
+cluster, which will complete the process.
+
+
+.. _free-trial-configure:
+
+Configure the free cluster
+==========================
+
+In the configuration wizard, the organization you created will automatically be
+selected already. All that remains is to choose a region where the cluster will
```

Comment:
Maybe "the organization you created will be pre-selected"? A bit less verbose...